### PR TITLE
Showing user details when user logs in

### PR DESCRIPTION
**Pull request by 06hpathak**

### CODE REVIEW

File: src/assets/images/users/profile avtar.jpg
Comment (amitad16) [inline src/assets/images/users/profile avtar.jpg:1]: you can rename it it `avatar_default`

File: src/layout/MainLayout/Header/HeaderContent/Profile/index.js [hunk 1]
```diff
@@ -27,9 +27,12 @@ import ProfileTab from './ProfileTab';
 import SettingTab from './SettingTab';
 
 // assets
-import avatar1 from 'assets/images/users/avatar-1.png';
+import avatar1 from 'assets/images/users/profile avtar.jpg';
```
Comment (amitad16) [inline src/layout/MainLayout/Header/HeaderContent/Profile/index.js:30]: rename the file name to `avatar_default`

Also, rename the variable
`import defaultAvatar from 'assets/images/users/avatar_default.jpg';`

**Review by amitad16 — COMMENTED** [review]
minor comments

File: src/assets/images/users/profile avtar.jpg
Comment (06hpathak) [reply src/assets/images/users/profile avtar.jpg:1]: done

File: src/layout/MainLayout/Header/HeaderContent/Profile/index.js (hunk 1 above)
Comment (06hpathak) [reply src/layout/MainLayout/Header/HeaderContent/Profile/index.js:30]: done

**Review by amitad16 — APPROVED** [review]
LGTM

**Pull request merged**